### DISCUSSION TITLE
Terminology issue regarding what is "immersive"

### DISCUSSION THREAD

**Discussion by Anix-x** [valinet/ExplorerPatcher#95] [General]
Currently the Readme.md says the program can "disable the immersive contex menu system-wide". What does it mean? It seems that in File Explorer it replaces the Win11-style context menus with Win10-style immersive menus or am I wrong? It seems it does not bring back the Win7/8 style context menus.

**Comment by valinet** [comment]
Yeah, idk how to call them, there are at least 20 different context menus, that's where this problem originally stems from...

The patcher is able to disable the new Windows 11 compact context menu thing. This one (which indeed I called "immersive"):
![image](https://user-images.githubusercontent.com/6503598/137412460-7d4cb190-c5e3-4c0a-803c-ecb6d5b0ae44.png)

And it reverts to this one:
![image](https://user-images.githubusercontent.com/6503598/137412513-c94307ce-26c0-424d-9640-97f2e6d4c9bd.png)

Now, idk, if you have a better terminology, I am happy to apply that. Windows 7/8 menus...? What do you mean?

**Reply by Tiagoquix** [reply]
@valinet The docs about registering as a shell extension mention that is it required to register EP as a shell extension to **disable** the immersive menus. However, in the app itself, the options reads: "**Use** immersive menus when displaying Windows 10 context menus". I think this is a bit confusing, since the option appears to disable the menus instead of enabling them.

**Comment by Anix-x** [comment]
Currently the program reverts to the Windows-10 style menus, not to Windows 8/7/XP/95 ones. So, I suggest "Enable Windows 10-style menus".
